release-note-script/src/main/java/MergeReleaseNotes.java

```diff
@@ -36,10 +36,12 @@ public class MergeReleaseNotes {
       new EnumMap<>(Edition.class);
 
   public static void main(String... args) throws Exception {
-    if (args.length == 0) {
-      System.err.printf(
-          "Usage: java %s.java scalardb.md cluster.md graphql.md sql.md%n",
-          MergeReleaseNotes.class.getSimpleName());
+
+    if (args.length > 0) {
+      if (args[0].equalsIgnoreCase("-h") || args[0].equalsIgnoreCase("--help")) {
```

Comment:
I think options are basically case-sensitive, so how about using `equals()` instead considering we would add some more options including `-H` (just in case) ?

Reply:
Thank you for pointing it out! I agree with you.
I fixed it at 4380207